pkg/cmd/grafana-cli/commands/install_command.go

```diff
@@ -233,6 +233,9 @@ func extractFiles(archiveFile string, pluginName string, filePath string, allowS
 				return fmt.Errorf(permissionsDeniedMessage, newFile)
 			}
 		} else {
+			// Create needed directories to extract file
+			os.MkdirAll(filepath.Dir(newFile), 0755)
```

Comment:
The error should be checked, linting fails at the moment because it isn't.

Reply:
Fixed, thanks!